app/src/main/java/no/usn/mob3000/ui/screens/user/ProfileEditScreen.kt

```diff
@@ -83,7 +84,8 @@ fun ProfileEditScreen(
                 value = displayName,
                 onValueChange = { displayName = it },
                 label = { Text(stringResource(R.string.profile_edit_display_name)) },
-                placeholder = { Text(stringResource(R.string.profile_edit_display_name_placeholder)) },
+                placeholder = { Text(stringResource(R.string.profile_edit_display_name_placeholder), color = MaterialTheme.colorScheme.onBackground.copy(alpha = 0.6f)) },
```

Comment:
Same as [with this comment](https://github.com/frigvid/mob3000-group11/pull/27/files#r1847357323). Since there are multiple parameters, it should be indented.